Zype/Categories/UIView+UIView_CustomizeTheme.m

```diff
@@ -26,7 +26,7 @@ - (void)borderCustomizeTheme {
 
 - (void)borderColorCustomizeTheme {
     self.layer.borderWidth = 1.0f;
-    self.layer.borderColor = (kAppColorLight) ? kLightTintColor.CGColor : kDarkTintColor.CGColor;
```

Comment:
Review comments and discussion thread https://zype.slack.com/archives/GGVNBSD4H/p1603349388043600